io.github.manisandro.gImageReader.yml

```diff
@@ -0,0 +1,77 @@
+app-id: io.github.manisandro.gImageReader
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: gimagereader-gtk
+rename-desktop-file: gimagereader-gtk.desktop
+rename-appdata-file: gimagereader-gtk.appdata.xml
+rename-icon: gimagereader
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --talk-name=org.a11y.Bus
+  - --own-name=org.gnome.gimagereader.*
+  - --talk-name=org.gtk.vfs.*
+  - --filesystem=xdg-run/gvfsd
+  - --env=TESSDATA_PREFIX=/app/share
+  # Filesystem access
+  # for Open
+  # for Add Image/Add another image
+  # for Add Folder
+  # for Import PDF
+  # for Save
+  # for Save as...
+  # for Export
+  # for Append Project
+  # for Export to ODT
+  - --filesystem=home
+  # Scanners access
+  - --device=all
+
+modules:
+  - name: gimagereader
+    buildsystem: cmake
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+      - /man
+      - /share/gtk-doc
+      - /share/man
+      - /share/pkgconfig
+      - /lib/*.la
+      - '*.a'
+    config-opts:
+      - -DCMAKE_INSTALL_PREFIX='/app'
+      - -DINTERFACE_TYPE=gtk
+      - -DENABLE_VERSIONCHECK=0
+    post-install:
+      # Use the SVG icon instead for clearity
+      - rm -r "${FLATPAK_DEST}/share/icons/hicolor"
+      - install -Dm644 data/icons/gimagereader.svg "${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/gimagereader.svg"
+    modules:
```

Comment:
splitting everything to so many is unmaintainable.

Reply:
no it is not. been there, done that. "copied from other" is a fallacy.

Reply:
yes and build in order (ie the top level module list).

valid exceptions are shared modules and auto-generated manifests.

Reply:
right now it doesn't track either.